bokeh/core/properties.py

```diff
@@ -918,7 +919,7 @@ def _repr_pretty_(self, p, cycle):
         else:
             with p.group(4, '%s(' % name, ')'):
                 props = self.properties_with_values().items()
-                sorted_props = sorted(props, key=lambda (prop, _): prop)
+                sorted_props = sorted(props, key=itemgetter(0))
```

Comment:
This is just great improvement. The alternative `lambda arg: arg[0]` isn't any better. If a feature is taken away, presumably for good reasons, there should be given something in return, maybe pattern matching? Even if such thing was introduced, I doubt it would be an expression, so it wouldn't fit into `lambda` anyway.